apps/sms/views/shared/style/panel.css

```diff
@@ -0,0 +1,65 @@
+.panel {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+
+  display: none;
+
+  overflow: auto;
+
+  background: white;
+
+  animation: .2s both linear;
```

Comment:
only thing that's easy to miss here is the `both` : it means the style before/after the animation are kept as they are at the start/end of the animation. TBH I don't always get how this works but I thought it was better with it ;)